### DISCUSSION TITLE
人生第一次進去喜來登

### DISCUSSION THREAD

**Discussion by THH1118** [1101-GS4719-C/Python-Community-GS4719-C#1671] [General]
前陣子報名了Ansys舉辦的台灣用戶技術大會，這個活動主要有以下幾個部分:
(1)	主辦方邀請了很多大佬來分享一些國際趨勢，然後要怎麼透過Ansys的軟體去模擬，今年有六大主題，有講一些無人載具、半導體……等，從早上到晚上5點大概每一個多小時每個主題就會有一場演講。
(2)	除了場內的演講，場外也會有很多科技業的攤販，其實就很像學校辦的企業博覽會，聽他們介紹之餘還可以拿一些小獎品。

總體來說還是蠻推薦參加這種活動的，雖然比較多是聽不懂的狀況，因為那裏分享的都是比較專業的資訊，去參加的大部分也都是已經出社會工作的人，所以對學生來講會聽不懂也正常，但是場內場外其實有很多接待的人員也都是專業的，他們其實很樂於講解，像我有一場演講因為場內太多人在聽所以沒辦法進去，結果接待人員就很用心的跟我講解了這場演講大概的內容(而且是淺顯易懂的講法)。而且因為參加了這個活動直接解鎖在喜來登吃自助餐的成就，虛榮++。

**Comment by 1chooo** [comment]
我覺得每次去參加類似這種大會又或是 Summit 都會讓自己的見識增加，不只可以聽到一些很厲害的人分享，我覺得還有一個很重要的就是可以遇到同領域的人，同樣興趣的人遇到一起討論總會產生新的火花，像是我在 AWS 就會遇到一些在 Linkedin 先加過的人，有種網路上看到很厲害的人出現在眼前，又能聊上天的感覺，其實挺不賴的！

最後推薦大家歡迎多來參加 AWS 在台灣舉辦的工作坊，又或是關注我們之後舉辦的活動！
